FranzKrekeler/vinecopulaslab/requirements.txt

```diff
@@ -0,0 +1,6 @@
+numpy
```

Comment:
Nice, thank you for including this, however you need to add the version numbers.